recipe/activate.sh

```diff
@@ -1,2 +1,6 @@
 export OCAMLLIB=$CONDA_PREFIX/lib/ocaml
 export OCAML_PREFIX=$CONDA_PREFIX
+
+head -2 ${CONDA_PREFIX}/lib/ocaml/runtime-launch-info
+sed -i "1s#/opt/anaconda1anaconda2anaconda3#${CONDA_PREFIX}#" ${CONDA_PREFIX}/lib/ocaml/runtime-launch-info
```

Comment:
Why is there a need to do this change every time the env is activated?

Reply:
The issue here is that the relocation is not working to the `runtime-launch-info`. So the solution I use is to have a recognizable string that I can switch in the `activate.sh` when the PREFIX is known.

Reply:
Note that the failure related to `runtime-launch-info` occurs when trying to build `ocamlfind`, it uses the information in the file to find ocaml lib and the error ends up being '/home/.../<timestamp of the ocaml local build>/bin/ocamlrun' not found. It ends-up having the hard-code timestamp of when I build ocaml. I tried various things without success

Reply:
Note that the failure related to `runtime-launch-info` occurs when trying to build `ocamlfind`, it uses the information in the file to find ocaml lib and the error ends up being '/home/.../<timestamp of the ocaml local build>/bin/ocamlrun' not found. It ends-up having the hard-code timestamp of when I build ocaml. I tried various things without success

Reply:
I used this `/opt/anaconda1anaconda2anaconda3` since it is also recognized for relocation. The in-recipe test does not catch this issue since it does not use `runtime-launch-info` and I don't know enough of ocaml to design one

Reply:
@wolfv It'd be great if you had sometime to look into this. We have a build that hardcode the built-time PREFIX in a shell script with binary data. I did try the:
```
  prefix_detection:
    ignore_binary_files: true
    force_file_type:
      text: lib/ocaml/runtime-launch-info
```
But that does not seem to work and I seemed to have to use the dirty trick above. There is not a way to test this directly in the recipe, I found it when using the local package to build `ocamlfind`

@danielnachun Given that most of the recipes related `ocaml` I worked on comes from your channel, you may have ideas as to how to solve this issue